packages/autocomplete-js/src/__tests__/reverseHighlightHit.test.ts

```diff
@@ -0,0 +1,109 @@
+import { reverseHighlightHit } from '../highlight';
+
+describe('reverseHighlightHit', () => {
+  test('returns a reversed partially highlighted Recent Search hit', () => {
+    expect(
+      reverseHighlightHit({
+        hit: {
+          _highlightResult: {
+            query: {
+              value:
+                '__aa-highlight__amazon ((fire__/aa-highlight__ tv)) tablet??',
+            },
+          },
+        } as any,
```

Comment:
If you add an `objectID`, you don't need to cast it to `any`.

Reply:
I forgot to add the `query` in the hit, fixed in https://github.com/algolia/autocomplete.js/pull/388/commits/e59e4d4b483077d35aa6b6df44094d35098b9d7a